h/api/storage.py

```diff
@@ -33,15 +34,15 @@ def annotation_from_dict(data):
     return models.elastic.Annotation(data)
 
 
-def fetch_annotation(request, id, _postgres=None):
+def fetch_annotation(request, annotation_id, _postgres=None):
```

Comment:
Is there a particular reason for this change? What kind of id would you pass into a function called `fetch_annotation` other than an annotation id?